packages/dom/src/dom/get-scroll-container.js

```diff
@@ -6,13 +6,13 @@ import getComputedStyle from './get-computed-style';
 /**
  * Given a DOM node, finds the closest scrollable container node.
  *
- * @param {Element} node Node from which to start.
+ * @param {Element | null} node Node from which to start.
  *
- * @return {?Element} Scrollable container node, if found.
+ * @return {undefined | Element} Scrollable container node, if found.
```

Comment:
Nit: I generally prefer `X | undefined` instead of `undefined | X`. Or is there a reason to prefer the latter?